authorize/evaluator/opa/functions.go

```diff
@@ -6,6 +6,9 @@ import (
 	"fmt"
 
 	lru "github.com/hashicorp/golang-lru"
+
+	_ "github.com/pomerium/pomerium/authorize/evaluator/opa/policy" // load static assets
```

Comment:
Why did this get added here? 🤔

Reply:
👍  bad conflict resolution on my part.  fixed.